toxcore/ccompat.h

```diff
@@ -42,6 +42,12 @@
 #define nullptr NULL
 #endif
 
+#ifdef _MSC_VER
+// MSVC has a header that disallows the above nullptr thing. Defining the
+// following symbol disables that check.
+#define _ALLOW_KEYWORD_MACROS
```

Comment:
Please move this to the copts so it's always defined before any headers are included.

Reply:
Done.